src/util/inject.js

```diff
@@ -4,9 +4,11 @@ const AsyncFunction = Object.getPrototypeOf(async function () {}).constructor;
 /**
  * @param {Function} func - Function to run in the page context (can be async)
  * @param {Array} args - Array of arguments to pass to the function via spread
```

Comment:
This isn't exactly to do with the changes in this PR, but reviewing it made me realise that `args` is optional too:

```suggestion
 * @param {Array} [args] - Array of arguments to pass to the function via spread
```

I'm not suggesting to pass `undefined` instead of `[]` anywhere, but it would (accurately) denote that an `inject()` call only needs `func` in order to work... though I can't think of any use-cases for that myself.

Reply:
Indeed, we do use that: for getting the css map and language data!

Reply:
Oops, yeah.

I am not awake enough.